package.json

```diff
@@ -211,20 +212,41 @@
     }
   },
   "resolutions": {
-    "@rmwc/textfield": "link:./packages/ui/rmwc/textfield",
-    "@rmwc/base": "link:./packages/ui/rmwc/base",
+    "tslib": "2.4.0",
     "@types/react": "16.14.2",
+    "typescript": "4.5.5",
     "@typescript/lib-dom": "npm:@types/web@*",
     "@types/webpack": "^5.28.0",
     "react": "16.14.0",
     "react-dom": "16.14.0",
     "pretty-format": "25.5.0",
-    "typescript": "4.5.5",
     "codex-tooltip": "1.0.2",
     "@pulumi/pulumi": "3.25.0",
     "@pulumi/aws": "4.28.0",
     "@types/eslint": "8.2.1",
-    "yargs": "^17.3.1"
+    "yargs": "^17.3.1",
+    "@material/base": "^5.1.0",
+    "@material/dom": "^5.1.0",
+    "@material/textfield": "^5.1.0",
+    "@material/button": "^5.1.0",
+    "@material/checkbox": "^5.1.0",
+    "@material/chips": "^5.1.0",
+    "@material/data-table": "^5.1.0",
+    "@material/dialog": "^5.1.0",
+    "@material/drawer": "^5.1.0",
+    "@material/elevation": "^5.1.0",
+    "@material/fab": "^5.1.0",
+    "@material/floating-label": "^5.1.0",
+    "@material/form-field": "^5.1.0",
+    "@material/icon-button": "^5.1.0",
+    "@material/linear-ripple": "^5.1.0",
+    "@material/list": "^5.1.0",
+    "@material/radio": "^5.1.0",
+    "@material/select": "^5.1.0",
+    "@material/snackbar": "^5.1.0",
+    "@material/switch": "^5.1.0",
+    "@material/tab-scroller": "^5.1.0",
+    "@material/top-app-bar": "^5.1.0"
```

Comment:
This is the wrong place for these dependencies. They should be moved to the package that's actually using these. If we're using RMWC, I'm not sure why we need to import these as dependencies at all. We can discuss this over a call if you're unsure what to do with these.